lib/reporting/identity_verification_report.rb

```diff
@@ -104,8 +110,12 @@ def as_csv
       csv << ['Workflow completed - Verified', idv_final_resolution_verified]
       csv << ['Workflow completed - Total Pending', idv_final_resolution_total_pending]
       csv << ['Workflow completed - GPO Pending', idv_final_resolution_gpo]
```

Comment:
Currently these pending attributes include people who were flagged for fraud review. We want to exclude them and have them only included in the "Fraud review" values directly below.